src/attachments/imageResizer.spec.js

```diff
@@ -118,6 +123,25 @@ describe("reduceImage", () => {
     };
   }
 
+  it("can minify and trim transparent pixels from all edges (png)", test009("png"));
+  it("can minify and trim pixels from all edges (jpg)", test009("jpg"));
+  function test009(extension) {
```

Comment:
[SUGGESTION] Wenn der frühere Vorschlag zum Refactoring / Extrahieren in `modifyImage.spec.js` angenommen wird, gerne auch hier das Gleiche.

Reply:
Das ist für mich jetzt "Out of scope" für diesen PR :)